WtWSMS/history/characters/_LM_new_characters.txt

```diff
@@ -2320,6 +2381,14 @@ HeruliChar102 = {
 }
 
 #Vandals
+1900077_father = {
+	name = "Wisimar" # PLRE vol. 1 Visimar
```

Comment:
He should be named "Visimar" instead of "Wisimar", that's how I spelled this name in my new Vandal namelist.

Reply:
fixed for next push!